third_party/move/move-vm/types/src/unit_tests/test_value_to_identifier_mapping.rs

```diff
@@ -0,0 +1,163 @@
+// // Copyright (c) The Move Contributors
+// // SPDX-License-Identifier: Apache-2.0
+//
+// use crate::values::{SizedID, Struct, Value};
```

Comment:
This entire file is commented. We don't need it anymore?

Reply:
Uncomented, and brought back all the tests :)

Reply:
Yes, this is stale and a pretty bad test which is no longer relevant because we have new native value impls _ sizes. I would rather prune it and find a better way to implement this logic.